### ISSUE TITLE
import oracledb broken the electron app on get NODE_MODULE_VERSION error

### ISSUE THREAD

**Issue by anuxs** [SimulatedGREG/electron-vue#167]
### Found an issue or bug? Tell me all about!
when npm install oracledb from https://github.com/oracle/node-oracledb ,install is ok , can connect oracle db in a separated single npm script file , no error there, but use oracle db in electron-vue app ,get NODE_MODULE_VERSION error as below :

##### Describe the issue / bug.
Uncaught Error: The module '/Users/langge/dev/test/h5js/webtest/electron-vue/app/node_modules/oracledb/build/Release/oracledb.node'
was compiled against a different Node.js version **using
NODE_MODULE_VERSION 51. This version of Node.js requires
NODE_MODULE_VERSION 53.** Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or`npm install`).
    at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:173:20)
    at Object.Module._extensions..node (module.js:598:18)
    at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:173:20)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/langge/dev/test/h5js/webtest/electron-vue/app/node_modules/oracledb/lib/oracledb.js:35:19)
    at Object.<anonymous> (/Users/langge/dev/test/h5js/webtest/electron-vue/app/node_modules/oracledb/lib/oracledb.js:405:3)
##### How can I reproduce this problem?
see the code :
```
<template>
  <div>
    <p><button v-on:click="dbClick">Query Oracle</button>{{dbtext}}</p>
  </div>
</template>

<script>
  import os from 'os'
  import oracledb from 'oracledb'

   export default {
    data () {
      return {
        dbtext: '查询结果'
      }
    },
    methods : {
          dbClick : function () {
            this.dbtext=''
            oracledb.getConnection({
              user: "ncmyl",
              password: "ncmyl",
              connectString: "192.168.31.221:1521/orcl"
            },
            (err, connection) => {
              if (err) {
                console.error(err.message);
                return;
              }
              connection.execute(
                "select cp.pk_corp,cp.unitcode,cp.unitname,cp.isseal " +
                " from bd_corp cp" +
                " where cp.isseal like :isseal" +
                " order by cp.unitcode " ,
                ['N'],
                (err, result) => {
                  if (err) {
                    console.error(err.message);
                      connection.close();
                    return;
                  }
                  console.log(result.metaData); 
                  console.log(result.rows); 
                  connection.close();
                });
            });
            console.log(this.dbtext)
            }
        }
  }
</script>
```

##### If visual, provide a screenshot.
\#

##### Tell me about your development environment.
  * Node version:v7.7.4
  * NPM version:4.1.2
  * vue-cli version: (if necessary)2.8.1
  * Operating System:macOS

**Comment by anuxs** [comment]
I have resolved this problem 。Uninstall node 7.7.4 , install node 8.0 nightly-build ,can run .

beacuse node 7.7.4 = NODE_MODULE_VERSION 51 ; node 8.0 = NODE_MODULE_VERSION 53

install node 8.0 ,reinstall oracledb and rebuild it in app , run ok .

**Comment by SimulatedGREG** [comment]
This may be a case where the native node module (those written in C, and not purely JS) has to be built against the electron runtime to function properly. This process isn't always simple, but things are usually better when using `electron-builder` since the build step will ensure things are built properly. Glad you got this sorted out.
